src/locale/pages/organizeCampaigns/en.yml

```diff
@@ -14,3 +14,5 @@ feedback:
     heading: Feedback and Surveys (none configured)
     copy: Collecting feedback (during phonebanks or standalone) can help you work more efficiently
     create: Create survey
+campaignManager: Campaign Manager
+noCampaignManager: No campaign manager
```

Comment:
This replaces `noManager`, but `noManager` is still in the file.

Reply:
Good catch!